### DISCUSSION TITLE
KeyDB as an alternative to Redis

### DISCUSSION THREAD

**Discussion by wernerreuser** [juicedata/juicefs#2817] [Ideas]
I was wondering if KeyDB would be a valid alternative to Redis as it has replication / ha by default and is fairly easy to setup and maintain. 

It is Redis compatible in means of interface and has transactions: https://docs.keydb.dev/docs/transactions/

**Comment by xiaogaozi** [comment]
Please refer to this document: https://juicefs.com/docs/community/databases_for_metadata#keydb

**Reply by wernerreuser** [reply]
Thanks !
